cfme/tests/services/test_request.py

```diff
@@ -18,7 +17,7 @@
 ]
 
 
-def test_copy_request(appliance, setup_provider, provider, catalog_item, request):
+def test_copy_request_bz1194479(appliance, setup_provider, provider, catalog_item, request):
```

Comment:
I wonder if this naming convention when automating tests for BZ's should be part of our standard (i.e. test_$blah_bz$bznum)